### ISSUE TITLE
[Android] onDeepLink result data is a string instead of an object.

### ISSUE THREAD

**Issue by curiousdustin** [AppsFlyerSDK/appsflyer-react-native-plugin#242]
# Report

## Plugin Version

6.1.41

## On what Platform are you having the issue?

Android

## What did you do?

Add callback to `onDeepLink`.

## What did you expect to happen?

I expect the callback to be called with a `result` object that has a `data` property that is also an object.

(just like on iOS, and also to be consistent with other callbacks that have a `result` with `data`.

## What happened instead?

The type of `data` is `string` instead.

## Please provide any other relevant information.

``` js
res => { console.log(typeof res.data) } // object on iOS, string on Android
```

Perhaps the solution is to just remove `toString()` here? https://github.com/AppsFlyerSDK/appsflyer-react-native-plugin/blob/afd390eeba54fbbb5fd3780827495d5be4601a8c/android/src/main/java/com/appsflyer/reactnative/RNAppsFlyerModule.java#L180

**Comment by amit-kremer93** [comment]
@curiousdustin thank you. will be fixed next release

**Comment by amit-kremer93** [comment]
Fixed with the latest version (6.2.10)
